CMakeLists.txt

```diff
@@ -34,6 +34,9 @@ else()
   message(STATUS "Building in ${CMAKE_BUILD_TYPE} configuration")
 endif()
 
+include(FindOpenACC)
```

Comment:
Are we using OpenACC?

Reply:
We don't. It would simplify the grid filter, but the compiler is not part of the mainstream CUDA SDK yet. It should not be in the PR.